src/NuGet.Core/NuGet.Commands/RestoreCommand/Diagnostics/UnresolvedMessages.cs

```diff
@@ -25,20 +25,48 @@ public static class UnresolvedMessages
         /// <summary>
         /// Log errors for missing dependencies.
         /// </summary>
-        public static async Task LogAsync(IEnumerable<IRestoreTargetGraph> graphs, RemoteWalkContext context, ILogger logger, CancellationToken token)
+        public static async Task LogAsync(
+            IEnumerable<IRestoreTargetGraph> graphs,
+            IList<IRemoteDependencyProvider> remoteLibraryProviders,
+            SourceCacheContext sourceCacheContext,
```

Comment:
do you really need to change this api? you don't need these for packageRef unresolved messages.

Reply:
This specific one is just a formatting change, for readability. 

I can change it back. 
It's the internals of it that are changing.